specification/common-types/data-plane/v1/types.json

```diff
@@ -70,5 +77,40 @@
       },
       "description": "The resource management error additional info."
     }
+  },
+  "responses": {
+    "ErrorResponse": {
+      "description": "Error response for all Azure services to return error details for failed operations.",
+      "headers": {
+        "x-ms-error-code": {
+          "type": "string",
+          "description": "The error code"
+        }
+      },
+      "schema": {
+        "$ref": "#/definitions/ErrorResponse"
+      }
+    }
+  },
+  "parameters": {
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for this operation.",
+      "minLength": 1
+    }
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
```

Comment:
Need to investigate what values for authorizationUrl need to be supported.  Could be multiple / many.

Reply:
This isn’t definitive, but a search of the latest versions of all the API docs in this repo shows only one `authorizationUrl`:
```
>ls -d specification/*/resource-manager/**/stable | while read d; do ls -dT $d/* | tail -1; done | while read d; do grep -h '"authorizationUrl"' $d/*.json; done | sort | uniq -c
 391       "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
```